### DISCUSSION TITLE
The UK city of Bangor is where?

### DISCUSSION THREAD

**Discussion by ParanoidUser** [ParanoidUser/yolo#362] [Galaxy Trivia] [answered]
**Hint:** 🐲

**Comment by peterbaccaro** [comment, marked as answer]
The UK city of Bangor is a small coastal city located in North Wales. It is situated on the Menai Strait, which separates the island of Anglesey from the Welsh mainland.

**Comment by skzidan** [comment]
Bangor (English: /ˈbæŋɡər/; Welsh: [ˈbaŋɡɔr] ( listen)) is a cathedral city and community in Gwynedd, North Wales. It is the oldest city in Wales. Historically part of Caernarfonshire, it had a population of 18,322 in 2019, according to the Office for National Statistics.
